cmake/modules/FindFFMPEG.cmake

```diff
@@ -39,19 +39,8 @@ macro(buildFFMPEG)
 
   set(MODULE_LC ffmpeg)
 
-  # We require this due to the odd nature of github URL's compared to our other tarball
```

Comment:
Is it possible to keep using the URL field for pointing the buildsystem to a custom git repo location? Usually it's easier to test a few patches on top of ffmpeg this way

Reply:
You can still use the `-DFFMPEG_URL=` option, This just removes the overrides we require for it because we were using the github archive url.